[2024-10-14T21:22:20.723+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [queued]>
[2024-10-14T21:22:20.732+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [queued]>
[2024-10-14T21:22:20.732+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-10-14T21:22:20.744+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): Variables.set_openweather_api> on 2024-10-14 20:00:00+00:00
[2024-10-14T21:22:20.752+0000] {standard_task_runner.py:60} INFO - Started process 220 to run task
[2024-10-14T21:22:20.754+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_get_weather_data', 'Variables.set_openweather_api', 'scheduled__2024-10-14T20:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag_get_weather.py', '--cfg-path', '/tmp/tmpf2rx5jlf']
[2024-10-14T21:22:20.756+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask Variables.set_openweather_api
[2024-10-14T21:22:20.768+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-10-14T21:22:20.808+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [running]> on host cc88bfa9cc01
[2024-10-14T21:22:20.899+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_get_weather_data' AIRFLOW_CTX_TASK_ID='Variables.set_openweather_api' AIRFLOW_CTX_EXECUTION_DATE='2024-10-14T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-14T20:00:00+00:00'
[2024-10-14T21:22:20.908+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-10-14T21:22:20.912+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-10-14T21:22:20.920+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_get_weather_data, task_id=Variables.set_openweather_api, execution_date=20241014T200000, start_date=20241014T212220, end_date=20241014T212220
[2024-10-14T21:22:20.966+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-14T21:22:21.002+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-14T21:51:19.390+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [queued]>
[2024-10-14T21:51:19.396+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [queued]>
[2024-10-14T21:51:19.396+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-10-14T21:51:19.403+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): Variables.set_openweather_api> on 2024-10-14 20:00:00+00:00
[2024-10-14T21:51:19.410+0000] {standard_task_runner.py:60} INFO - Started process 133 to run task
[2024-10-14T21:51:19.412+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_get_weather_data', 'Variables.set_openweather_api', 'scheduled__2024-10-14T20:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_get_weather.py', '--cfg-path', '/tmp/tmp6_lbiksl']
[2024-10-14T21:51:19.413+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask Variables.set_openweather_api
[2024-10-14T21:51:19.422+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-10-14T21:51:19.444+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [running]> on host fd92e3be397d
[2024-10-14T21:51:19.489+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_get_weather_data' AIRFLOW_CTX_TASK_ID='Variables.set_openweather_api' AIRFLOW_CTX_EXECUTION_DATE='2024-10-14T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-14T20:00:00+00:00'
[2024-10-14T21:51:19.494+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-10-14T21:51:19.497+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-10-14T21:51:19.502+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_get_weather_data, task_id=Variables.set_openweather_api, execution_date=20241014T200000, start_date=20241014T215119, end_date=20241014T215119
[2024-10-14T21:51:19.544+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-14T21:51:19.577+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-14T21:59:22.999+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [queued]>
[2024-10-14T21:59:23.005+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [queued]>
[2024-10-14T21:59:23.006+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-10-14T21:59:23.015+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): Variables.set_openweather_api> on 2024-10-14 20:00:00+00:00
[2024-10-14T21:59:23.026+0000] {standard_task_runner.py:60} INFO - Started process 141 to run task
[2024-10-14T21:59:23.030+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_get_weather_data', 'Variables.set_openweather_api', 'scheduled__2024-10-14T20:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_get_weather.py', '--cfg-path', '/tmp/tmp90jk6q9q']
[2024-10-14T21:59:23.032+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask Variables.set_openweather_api
[2024-10-14T21:59:23.047+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-10-14T21:59:23.082+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_get_weather_data.Variables.set_openweather_api scheduled__2024-10-14T20:00:00+00:00 [running]> on host ee2b315f6b67
[2024-10-14T21:59:23.149+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_get_weather_data' AIRFLOW_CTX_TASK_ID='Variables.set_openweather_api' AIRFLOW_CTX_EXECUTION_DATE='2024-10-14T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-14T20:00:00+00:00'
[2024-10-14T21:59:23.158+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-10-14T21:59:23.162+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-10-14T21:59:23.170+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_get_weather_data, task_id=Variables.set_openweather_api, execution_date=20241014T200000, start_date=20241014T215922, end_date=20241014T215923
[2024-10-14T21:59:23.202+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-14T21:59:23.223+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
